buildenv/jenkins/jobs/infrastructure/Bootjdk_Cleanup.groovy

```diff
@@ -0,0 +1,59 @@
+/*******************************************************************************
+ * Copyright IBM Corp. and others 2025
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] https://openjdk.org/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0 OR GPL-2.0-only WITH OpenJDK-assembly-exception-1.0
+ *******************************************************************************/
+
+// Remove all bootjdks from build machines.
+// Trigger initially setup for every 6 months.
+// Fire downstream job "all-nodes-matching-labels" with various params in parallel.
+
+ def defaultLocation = '$HOME/bootjdks'
+ def slackChannel = params.SLACK_CHANNEL ?: '#jenkins-sandbox'
+
+// Do not run on z/OS.
+def GROUPS = [[name:"Mac", labels:"ci.role.build&&sw.os.mac", location:defaultLocation],
+    [name:"Windows", labels:"ci.role.build&&sw.os.windows", location:defaultLocation],
+    [name:"AIX", labels:"ci.role.build&&sw.os.aix", location:"/opt/bootjdks"],
+    [name:"xlinux", labels:"ci.role.build&&sw.os.linux&&hw.arch.x86", location:defaultLocation],
+    [name:"plinux", labels:"ci.role.build&&sw.os.linux&&hw.arch.ppc64le", location:defaultLocation],
+    [name:"zlinux", labels:"ci.role.build&&sw.os.linux&&hw.arch.s390x", location:defaultLocation],
+    [name:"alinux", labels:"ci.role.build&&sw.os.linux&&hw.arch.aarch64", location:defaultLocation]
```

Comment:
Is there a reason to use this order instead of alphabetically by group name?
Is `defaultLocation` expanded in each of these to `$HOME/bootjdks`?

Reply:
No reason for the order. My only preference might be to keep the linuxes grouped together.
Yes, default location will be HOME/bootjdks and be resolved on the end nodes.

Reply:
Perhaps we can have a little more order: AIX, a (sorted) Linux group, Windows.

Reply:
Updated.